po/ru.po

```diff
@@ -14738,7 +14737,7 @@ msgstr "дней."
 #, fuzzy
```

Comment:
You should remove this line, if "Каждые" instead of "Every" should be displayed.
https://wiki.gnucash.org/wiki/Translation#Gettext_source_.28.po.29_file_format

Can you review the msgctxt "Daily"..msgctxt "Weekly" section, which fuzzy flags are obsolete?
If you like, you can also scan the whole file. I count 38 occurrences.